### DISCUSSION TITLE
Saving history while switching layouts in studio

### DISCUSSION THREAD

**Discussion by beaucolley** [foxglove/community#52] [Q&A]
I've got a couple of layouts setup in the studio app but when ever I switch perspectives all of the historical data in the graphs is lost. Is there a way all perspectives can be running at the same time?

**Comment by jtbandes** [comment]
Can you please share some more information about your setup and steps to reproduce the issue? What kind of connection/file are you using to view data? Do you experience the same problem if you use a Tab panel rather than separate layouts?

It is intentional that Studio only subscribes to and processes data in panels that are being actively used. Processing all data at all times would be untenable for most users.
